test_project/models.py

```diff
@@ -1 +1,30 @@
-from django.db import models
\ No newline at end of file
+from django.db import models
+from djangoplicity.newsletters.mailers import MailmanMailerPlugin, MailChimpMailerPlugin
+
+class SimpleMailer( MailmanMailerPlugin ):
+    name = 'Standard mailer'
+    # action_run_test = ''
+
+    action_parameters = [
+            ('name', 'list name', 'str'),
+            ( 'password', 'Admin password for list', 'str' ),
+            ( 'somenum', 'Some num', 'int' ),
+        ]
+    abstract = True
+
+    def __init__(self, *args, **kwargs):
+        pass
```

Comment:
Did you copy this from another location?

Reply:
yes, i did make this code in djangoplicity-actions. i used this because the mailer model  register plugins the same way mode that the actions model